### DISCUSSION TITLE
Filter out static content such as images?

### DISCUSSION THREAD

**Discussion by 0xdef1ant** [projectdiscovery/httpx#998] [Q&A]
What is the best way to filter out static assets such as images? I have a giant list of urls and want to filter out ones that resolve to serving .jpg, .gif, etc. The list of URLs don't necessarily contain the strings `.jpg`, `.gif`, etc, so it's not just a simple regex or `grep -v` to get rid of them

**Comment by olearycrew** [comment]
If images are coming with a non-standard file extension, I'm not sure how httpx could detect them in any other way...

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
